### DISCUSSION TITLE
Version 3.11

### DISCUSSION THREAD

**Discussion by lowlighter** [lowlighter/metrics#357] [Announcements]
## 📦 New features

* ↔️ **Introducing columns display size** <sup>✨ new!</sup>
  * #371 You can now use `columns` in `config_display` option for a 2-columns layout which will render as a single column on smaller devices
  * <img src="https://user-images.githubusercontent.com/22963968/121562803-19752480-ca1a-11eb-9062-1fabba1528c9.png" width="600">
* 🈷️ **Most used languages**
  * 5120f64 Estimation now display the last day that metrics was able to fetch from GitHub events, instead of the option value to avoid confusion
  * 94539fa Add `plugin_languages_aliases` to support custom languages display names
  * #356 Recently used languages now reproduce a "partial repository" from events fetched and retrieved the `.gitattributes` of your repository to take into account linguist overrides
  * #380 Indepth now display number of commits and files processed
  * * c13582ec Improvement in indepth algorithm to handle large `git diff`
* 💡 **Coding habits**
  * afdab27 Header now display how many commits events were used to compute habits
  * #373 Add `plugin_habits_trim` to trim unused hours on daily chart
* 🎭 **Comment reactions**
  * 7f08abb Add `plugin_reactions_limit_issues` to support custom number of issues and pull requests to fetch for reactions
    * `plugin_reactions_limit` now targets only comments from issues and pull requests 
* 🦑 **Miscelleanous**
  * aa4225f `config_timezone` now set the timezone of metrics action, meaning that all dates will be based on passed timezone (including data fetched from GitHub, like commits)
  * #364 Improve apt process in Dockerfile 
  * #372 Issue icons were replaced to match GitHub new design
  * #390 Upgrade to node 16 
  * 174f2a27  Add `delay` option which temporises jobs (can be used to avoid reaching abuse limit)
  * 78a7c97f Add `repositories_batch` which control how many repositories are fetched at once (can be useful for users with a really lot of activity)
  
## 🧰 Fixes
* 37f686a ✨ Fix a fatal error which prevented insights from loading
* 1a52951 🧱 Fix broken `config_output: json`
* #379 🌐 Generated workflow configurations had some options which were invalid
* 98e7f05 💡 Runtime errors are now displayed in output
* #393 ⏰ Fix several Wakatime display issues
* #354 🗃️ Use correct timezone in footer
* #400 🌟 Unknown licenses are now displayed gracefully
* 5a15dfd 👨‍💻📰 Make login case insensitive 
* #416 🈷️ Fix a case where undefiend file url made the plugin crash
  
## 💕 Sponsors and contributors
<img src="https://raw.githubusercontent.com/lowlighter/metrics/examples/metrics.sponsors.svg">
<img src="https://raw.githubusercontent.com/lowlighter/metrics/examples/metrics.contributors.svg">

[`♥️ Become a sponsor`](https://github.com/sponsors/lowlighter)

**Comment by Zenoo** [comment]
There should also be the `plugin_habits_trim` option in the habits plugin.
